client/my-sites/plugins/marketplace/util.ts

```diff
@@ -0,0 +1,17 @@
+//Navigation functions
+export function navigateToProductHomePage(
+	page: ( url: string ) => void,
+	selectedSiteSlug = '',
+	pluginSlug = 'wordpress-seo'
+): void {
+	page(
+		`/marketplace/product/details/${ pluginSlug }/${ selectedSiteSlug }?flags=marketplace-yoast`
```

Comment:
The `flags` query arg is generally used for testing and development only, and shouldn't be used in production. If we're leaving it here for testing purposes, let's add a comment to ensure that we've removed it before it goes to production.

Reply:
Ah yes since the testing effort is high on this feature thought of including the flag for ease. Will include a comment.